mass_editing/tests/test_mass_editing.py

```diff
@@ -49,8 +50,10 @@ def _create_user(self):
         })
 
     def _create_mass_editing(self, model, fields):
-        """Create a Mass Editing with Partner as model and
-        email field of partner."""
+        """
+        Create a Mass Editing with Partner as model and
+        email field of partner.
+        """
```

Comment:
Please do not change things just because yes 😉 

(even more when [this format isn't also the correct one](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings))